modules/rebalance-reference/pages/rebalance-reference.adoc

```diff
@@ -0,0 +1,345 @@
+
+[#reading-a-rebalance-report]
+== Reading a Rebalance Report
+
+Each rebalance report consists of a JSON document whose principal object is `stage_info`.
+This itself contains an object corresponding to each rebalance _stage_: one stage has occurred for each of the services deployed on the cluster.
+Therefore, if all six services have been deployed, six stages occur in a successful rebalance; and objects for `analytics`, `eventing`, `search`, `index` `query`, and `data` are provided.
+
+Among the details provided for each service are the times at which rebalance-processes started and ended, the durations of rebalance-processes, and the numbers of documents already handled and still to be handled.
+When rebalance concludes successfully, a report is duly generated, with all fields corresponding to the successful completion of the rebalance-processes they represent.
+In cases where rebalance is interrupted (for example, by the user's left-clicking on the *Stop Rebalance* button, in Couchbase Web Console, or due to auto-failover), a generated report will describe a partially unsuccessful rebalance; indicating, in certain fields, an incomplete or unstarted rebalance-process, by means of the value `false`.
+
+[#standard-fields]
+== Standard Fields
+
+Standard fields are provided for each stage.
+These are as follows:
+
+* `startTime`.
+The point in time at which the stage was started (or possibly, has been determined _not_ to have started).
+Specified as a UTC date/time string.
+
+* `completedTime`.
+The point in time at which the stage was completed (or possibly, has been determined _not_ to have completed).
+Specified as a UTC date/time string.
+
+* `timeTaken`.
+The amount of elapsed time since `startTime`.
+Specified as an integer, in milliseconds.
+The value is `undefined`, if the stage has been determined _not_ to have started.
+
+* `totalProgress`.
+The current, total progress that has been achieved for the stage.
+Specified as a percentage.
+In a completed report, the value is expected to be `100`.
+This is an _optional_ field, which may not appear if not implemented for the stage.
+
+* `perNodeProgress`.
+The current progress that has been achieved for the stage for each node.
+The value is an object, which itself provides a progress figure for each node.
+Each progress figure is specified as a floating-point number between `0.0` and `1.0`.
+In a completed report, the value is expected to be `1`.
+This is an _optional_ field, which may not appear if not implemented for the stage.
+
+* `subStage`.
+Details on any _substages_ that the stage may contain.
+For each _substage_, the standard fields are identical to those for each _stage_.
+This is an _optional_ field, which does not appear, if the stage contains no substages.
+(An example is _delta_recovery_, which is a substage reflected in the `data` output, when Delta Recovery has indeed been performed on one or more nodes running the Data Service.)
```

Comment:
"when Delta recovery has been performed" -> "when delta recovery Warmup has been performed" .

Unfortunately, delta recovery isn't very clearly understood by everyone hence specifying "warmup" is rather important word that needs to present.

Reply:
Done.